crypto/fipsmodule/bn/test/exp_tests.txt

```diff
@@ -0,0 +1,23 @@
+# Exp tests.
```

Comment:
You didn't delete bn_tests.txt after splitting up these files. Also don't forget to move https://github.com/dkostic/aws-lc/commit/11b50d39cf2378703a4ca6b6fee9d76a2e9852d1 to the new files.